corehq/tests/noseplugins/dividedwerun.py

```diff
@@ -91,21 +91,44 @@ def skip():
 
 
 def name_of(test):
+    """Returns the full name of the test as a string."""
     if not isinstance(test, ContextSuite):
         return str(test)
     context = test.context
     if isinstance(context, type) or isinstance(context, types.FunctionType):
         return context.__module__ + ":" + context.__name__
     if isinstance(context, types.ModuleType):
-        return context.__name__
+        return str(context.__name__)
```

Comment:
nit: I think this change is unnecessary because `context.__name__` is always a string. If not, the expression two lines above would raise on a non-string value (I don't think we've ever observed that).